configs/couplings/fesom-2.0-awicm-3.1+oifs-43r3-awicm-3.1.1+xios-2.5/fesom-2.0-awicm-3.1+oifs-43r3-awicm-3.1.1+xios-2.5.yaml

```diff
@@ -0,0 +1,11 @@
+components:
+- xios-2.5
+- rnfmap-awicm-3.1
+- oifs-43r3-awicm-3.1.1
+- fesom-2.0-awicm-3.1
+- oasis3mct-4.0-awicm-3.1
+coupling_changes:
+- sed -i '/COUPLENEMOECE = /s/.TRUE./.FALSE./g' oifs-43r3/src/ifs/module/yommcc.F90
+- sed -i '/COUPLEFESOM2 = /s/.FALSE./.TRUE./g' oifs-43r3/src/ifs/module/yommcc.F90
+- sed -i '/COUPLENEMOFOCI = /s/.TRUE./.FALSE./g' oifs-43r3/src/ifs/module/yommcc.F90
```

Comment:
Mmmmpf.....changing fortran via sed gives me a small headache. But we also change makefiles via sed.

So, at the end: hey, if it works, hooray 😄